### PR TITLE
change cb template to use dsc

### PR DESCRIPTION
**Pull request by YuanhengYang**
change cb template to use dsc

### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by yizhongwu** [comment]
The main change of the PR is
1. We using DSC instead CustomScriptExtension. It's more secure while passing password. 
2. Add one more machine for CM client
3. Remove dependency on AzCopy

File: sccm-currentbranch/azuredeploy.json [hunk 1]
```diff
@@ -90,19 +48,17 @@
...
-    "containerName": "[concat(toLower(parameters('prefix')), 'container')]",
-    "vnetId": "[resourceId(parameters('existingVnetResourcegroup'), 'Microsoft.Network/virtualNetworks', parameters('exsitingVnetName'))]",
-    "generalSettings": {
-      "storageAccountType": "Standard_LRS",
-      "diskstorageAccountType": "Premium_LRS",
-      "virtualMachinessize": "Standard_D2s_v3"
-    },
+    "dscScript": "DSC/DCConfiguration.zip",
+    "virtualNetworkName": "[concat(toLower(parameters('prefix')), '-vnet')]",
+    "vnetId": "[resourceId(resourceGroup().name, 'Microsoft.Network/virtualNetworks', variables('virtualNetworkName'))]",
```
Comment (bmoore-msft) [inline sccm-currentbranch/azuredeploy.json]: remove this - it's never used

File: sccm-currentbranch/azuredeploy.json [hunk 2]
```diff
@@ -90,19 +48,17 @@
...
+    "domainName": "contoso.com",
```
Comment (bmoore-msft) [inline sccm-currentbranch/azuredeploy.json]: seems like this should be a parameters (with defaultValue)?

Comment (yizhongwu) [reply sccm-currentbranch/azuredeploy.json]: It's for demo or lab purpose. We don't want user to customize it. So I put general name contoso.com here.

File: sccm-currentbranch/azuredeploy.json (hunk 1 above)
Comment (YuanhengYang) [reply sccm-currentbranch/azuredeploy.json]: Agreed

**Pull request merged**